internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp

```diff
@@ -74,15 +74,19 @@ void
 ExecPlanNodeVisitor::ExecuteExprNodeInternal(
     const std::shared_ptr<milvus::plan::PlanNode>& plannode,
     const milvus::segcore::SegmentInternalInterface* segment,
+    int64_t row_num,
```

Comment:
The variable "row_num" typically represents the line number. The naming of this variable could be improved in this context.

Reply:
still using active_count ?

Reply:
lgtm